### DISCUSSION TITLE
Nature Conferences Virtual Symposium: Spatial Biology

### DISCUSSION THREAD

**Discussion by zivy** [IBEXImagingCommunity/ibex_imaging_knowledge_base#220] [Announcements]
**When**: In 43 days, Tuesday Oct. 29, 2024.
**Who**: Dr. Andrea Radtke (Leica Microsystems), Dr. Rong Fan (Yale University), Dr. Adyary Fallarero (Thermo Fisher Scientific ), Dr. Chris Langsdorf (Thermo Fisher Scientific ), Dr. Leeat Keren (Weizmann Institute of Science), Dr. Long Cai (Caltech), Dr. Sarah Teichmann (Wellcome Sanger Institute), Dr. Xiao Wang (MIT), Dr. Bernd Bodenmiller (University of Zurich), Dr. Jeffrey Moffitt (Boston Children's Hospital).
**What**: Overview of key experimental and computational advances in spatial biology highlighting the transformative nature of spatial ‘omics techniques for biological discovery.
**Where**: Online (registration required).

For all the details and to register see the webinar announcement on the [Nature Conferences website](https://natureconferences.streamgo.live/spatial-biology/register?refer=naturecareers).

**Comment by DasEFX130** [comment]
Looking forward to it!

**Reply by radtkea** [reply]
Thanks so much!!! Me too. So grateful for the opportunity to showcase the great work of our community:)
